### PR TITLE
Add command history tests

### PR DESCRIPTION
**Pull request by benedictkhoomw**
## What this does
This PR:
1. Adds unit tests for various command history components
2. Adds unit tests for `MathUtil`

## How to test
1. Run all tests
2. Ensure they are passing

### CODE REVIEW

File: src/test/java/seedu/address/commons/util/MathUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        assertEquals(lower, MathUtil.clamp(lower, lower, upper));
+
+        // EP: Value below lower bound -> returns lower bound
+        assertEquals(lower, MathUtil.clamp(lower - 213, lower, upper));
+
+        // EP: Value above upper bound -> returns upper bound
+        assertEquals(upper, MathUtil.clamp(upper + 91, lower, upper));
+
+        // EP: Value strictly between bounds -> returns value
+        assertEquals(value, MathUtil.clamp(value, lower, upper));
```
Comment (stein414) [inline src/test/java/seedu/address/commons/util/MathUtilTest.java:34]: I think maybe adding 1 more test case value beyond the upper and lower bounds would make this more complete

File: src/test/java/seedu/address/logic/commandhistory/SuppliedCommandHistorySelectorTest.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+        // Check all entries in reverse
+        typicalSelector.navigateToOnePastLast();
+        for (int i = entries.size() - 1; i >= 0; i--) {
+            Optional<String> entryText = Optional.of(entries.get(i).value);
+            assertEquals(entryText, typicalSelector.selectPreviousUntilFirst());
+        }
+    }
+
+    @BeforeEach
```
Comment (stein414) [inline src/test/java/seedu/address/logic/commandhistory/SuppliedCommandHistorySelectorTest.java:103]: I think this should be at the top of the file to be clear what happens

**Review by stein414 — APPROVED** [review]
LGTM. Test passes and read through tests

File: src/test/java/seedu/address/commons/util/MathUtilTest.java (hunk 1 above)
Comment (benedictkhoomw) [reply src/test/java/seedu/address/commons/util/MathUtilTest.java:25]: Yup, there are above-upper-bound and below-lower-bound test cases

File: src/test/java/seedu/address/logic/commandhistory/SuppliedCommandHistorySelectorTest.java (hunk 2 above)
Comment (benedictkhoomw) [reply src/test/java/seedu/address/logic/commandhistory/SuppliedCommandHistorySelectorTest.java]: Ah good point. Thanks.

**Pull request merged**